security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -15,6 +15,28 @@ For an overview of the Compliance Operator, see xref:../../security/compliance_o
 
 To access the latest release, see xref:../../security/compliance_operator/compliance-operator-updating.adoc#olm-preparing-upgrade_compliance-operator-updating[Updating the Compliance Operator].
 
+[id="compliance-operator-release-notes-1-3-0"]
+== OpenShift Compliance Operator 1.3.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.3.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5102[RHBA-2023:5102 - OpenShift Compliance Operator enhancement update]
+
+[id="compliance-operator-1-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* Compliance Operator 1.3.0 now supports creation of the DISA-STIG profile within the supported profiles module for {product-title} 4.14. See https://docs.openshift.com/container-platform/4.13/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles
```

Comment:
We aren't "creating" the DISA-STIG profile, and the profile will be available on all supported branches (4.11+).  Additionally, 4.14 is not released content at this time so we shouldn't be mentioning it anywhere in our documentation. Try: 
"The DISA-STIG profile is now available for Compliance Operator 1.3.0. "

Additionally, you have a URL where there should be an xref.

Reply:
Fixed - Thank you!